rasa/nlu/featurizers/sparse_featurizer/lexical_syntactic_featurizer.py

```diff
@@ -290,12 +312,13 @@ def load(
         feature_to_idx_file = Path(model_dir) / f"{file_name}.feature_to_idx_dict.pkl"
         feature_to_idx_dict = io_utils.json_unpickle(feature_to_idx_file)
 
-        return LexicalSyntacticFeaturizer(meta, feature_to_idx_dict=feature_to_idx_dict)
+        return cls(meta, feature_to_idx_dict=feature_to_idx_dict)  # Fix for #10471
```

Comment:
```suggestion
        return cls(meta, feature_to_idx_dict=feature_to_idx_dict)
```
Think we don't need the comment within the code, the changelog should be enough evidence :)